### ISSUE TITLE
Bokeh Plot Not Rendering

### ISSUE THREAD

**Issue by gfbarbieri** [datapane/datapane#91]
### System Information

System Information
- OS: Windows 10
- Python version: 3.7.4 via Anaconda
- Python environment: conda
- Using jupyter: true
- Datapane version: 0.10.2

### Bug / Issue

I ran the following code (motivated from the Datapane examples). The Bokeh plot does not render in Google Chrome, but the table does render. This is not an issue for Matplotlib. Can this issue be reproduced?
```python
from bokeh.plotting import figure, output_file, show
import datapane as dp 
import numpy as np
import pandas as pd

df = pd.DataFrame(data=np.random.rand(10,3), columns=['one','two','three'])

bokeh_chart = figure()
bokeh_chart.circle([10,1],[10,1])

report = dp.Report(
    dp.Plot(bokeh_chart),
    dp.Table(df)
)

report.save(path='report.html', open=True)
```

**Comment by mands** [comment]
Hi @gfbarbieri - thanks for raising - yes we've noticed a mismatch between the python and JS versions of Bokeh which is causing this and will be deploying a fix to pip and conda in the next few days. Thanks again!

**Comment by mands** [comment]
Hi - please try the recently released 0.10.4 which should support bokeh again
